src/app/app.component.ts

```diff
@@ -14,9 +15,12 @@ import { UploadService } from './services/upload/upload.service';
 })
 export class AppComponent implements OnDestroy {
     public textareaFc: FormControl;
+    public Valid = Valid;
 
     private _sub: Subscription;
     private _fileSub: Subscription;
+    private _isRunValid: Valid | undefined;
```

Comment:
Ist genaugenommen eher sowas wie `_runValidationStatus`
```suggestion
    runValidationStatus: ValidationStatus | undefined;
```